CPT331.WebAPI/CPT331.WebAPI.csproj

```diff
@@ -143,11 +180,11 @@
   <ItemGroup>
     <Folder Include="App_Data\" />
     <Folder Include="Models\" />
-    <Folder Include="Scripts\" />
```

Comment:
This was probably "deleted" because the folder line isn't needed if there is content inside it.

Having scripts inside the folder likely negated the need for this line.

The csproj is maintained by Visual Studio, so I cannot provide an absolute answer behind these project files.